src/actions/common.go

```diff
@@ -58,17 +59,19 @@ func (es ExecSummary) Display() {
 				report.Brick.Name,
 				extools.Indent(report.Error.Error()))
 		case TAG_SKIP:
-			sb.WriteString(color.BlueString("SKIP  "))
+			sb.WriteString(color.BlueString("SKIP    "))
```

Comment:
Why not?
```suggestion
			sb.WriteString(color.BlueString("SKIP\t"))
```

Reply:
because we can manage the line length.
Later it could be good if we could display thing like that:

```
OK     infra-ground/envs/production/network
ERR    infra-ground/envs/production/bastion : lay has failed, output doesn't
         seem to has changed: exit with status 1
SKIP   infra-ground/envs/production/cluster_k8s
```

instead of that:

```
OK     infra-ground/envs/production/network
ERR    infra-ground/envs/production/bastion : lay has failed, output doesn't
seem to has changed: exit with status 1
SKIP   infra-ground/envs/production/cluster_k8s
```

Reply:
Ah je comprends l'idée. ça me paraît un peu ambitieux de gérer le wrapping de ligne nous même, de cette manière là, mais ça ne change rien fonctionnellement, donc :shrug: